cloud/billing/account-management.md

```diff
@@ -1,37 +1,59 @@
     the `Billing` details section.
-1.  Click `Add a payment method`. If you already have a payment method saved,
-    you need to delete it before you start. Click the trash can icon to delete
-    your saved credit card.
-1.  Type your credit card details, and click `Add billing address`.
-1.  Type your address, and click `Save payment method`.
-1.  Confirm your new credit card is saved by checking the details on
-    the `Billing` page.
+1.  In the `Wallet` section, click `Add payment method`. If you already have
+    three credit cards saved in the payment methods, you need to delete at least
+    one of them before you start. Click the trash can icon to delete your saved
+    credit card.
+1.  Type your credit card details, and click `Continue`.
+    The `Add payment method: billing address` page appears.
+1.  Type you billing address and company address. The company address is
+    required to determine any applicable sales tax.
+    * If your billing address is the same as your company address, select
+      `Billing  address is the same as company address`, and click `Save`.
+    * If your billing address is not the same as your company address, click
+      `Continue`. Type your company address, and click `Save`.        
+1.  Confirm your new credit card is saved by checking the details on the
+    `Billing` page.
+    You can update your billing address and company address by clicking the edit
+    icon in the `primary` payment method. However, you can only edit the billing
+    address on other payment methods.
 
 <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/add_credit_card.png" alt="Adding a payment method in Timescale Cloud"/>
 
+<highlight type="note">
+The first credit card that you add to the payment method is set as the `Primary
+payment method`, by default. After you add other cards to the payment method,
+click `Set as primary` to set that card as primary for payments.
+</highlight>
+
+</procedure>
+
 <highlight type="note">
 The first charge on your new credit card is for the period from the date you
 added the card, to the end of that calendar month. After that, all payment
 confirmations are for usage from the first of the month, to the last day of the
 month.
 </highlight>
 
-</procedure>
-
-## Add or change your billing email address
+## Add or update your billing email address
 By default, your Timescale Cloud payment confirmations are sent to the email
 address that you used when you signed up. This is also the address that receives
 alerts about your Timescale Cloud services. If you want your payment
-confirmations to also go to a different email address, for example the email address
-for your finance department, you can add it in the `Billing` details section.
+confirmations to also go to a different email address, for example the email
+address for your finance department, you can add it in the `Billing` details
+section.
 
 <highlight type="note">
```

Comment:
There are three note admonitions in a row here. Can we turn one or two into prose instead?

![image](https://user-images.githubusercontent.com/3914967/169724585-7694b00c-d2d8-4f91-a860-dfdf3ef28278.png)